### PR TITLE
Update the man page

### PR DESCRIPTION
**Pull request by bl-ue**
* Update copyright year and name
* Update repository link

### CODE REVIEW

**Review by sbrl — APPROVED** [review]
Looks ok to me. But I'm not an active contributor to the C client, and we'd want Arvid Gerstmann's express permission to change that (I don't know their @handle).

**Comment by bl-ue** [comment]
cc @Leandros

**Comment by bl-ue** [comment]
Yeah 😬 I forgot about that.  I just blindly changed it 😄

**Comment by sbrl** [comment]
I would suggest keeping their name there in the copyright notice and adding tldr-pages contributors, given that they were the original author.

**Review by owenvoke — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

File: man/tldr.1 [hunk 1]
```diff
@@ -37,12 +37,12 @@ render a local page for testing purposes
...
+.URL "https://github.com/tldr-pages/tldr-c-client" "" ""
 .SH REPORTING BUGS
 Report bugs through the Github repository:
-.URL "https://github.com/tldr-pages/tldr-cpp-client" "" "."
+.URL "https://github.com/tldr-pages/tldr-c-client" "" "."
 .SH COPYRIGHT
 The MIT License (MIT)
 
-Copyright (c) 2016 Arvid Gerstmann.
+Copyright (c) 2021 Arvid Gerstmann, the tldr-pages team and contributors.
```
Comment (mebeim) [inline man/tldr.1:47]:
```suggestion
Copyright (c) 2014-2021 Arvid Gerstmann, the tldr-pages team and contributors.
```

Copyright notices should cover all years in which the source and changes of it are subject to copyright (basically from the first year to the year of the last change), so in this case 2014-2021.

**Review by mebeim — CHANGES_REQUESTED** [review]

**Review by mebeim — APPROVED** [review]

**Review by sbrl — APPROVED** [review]

**Comment by bl-ue** [comment]
Can we merge now?

**Comment by sbrl** [comment]
@bl-ue No. We need the original author's express permission to change this.

**Review by marchersimon — APPROVED** [review]
Any reason this isn't merged yet?

File: man/tldr.1 [hunk 2]
```diff
@@ -37,12 +37,11 @@ render a local page for testing purposes
...
 .SH REPORTING BUGS
 Report bugs through the Github repository:
-.URL "https://github.com/tldr-pages/tldr-cpp-client" "" "."
+.URL "https://github.com/tldr-pages/tldr-c-client" "" "."
 .SH COPYRIGHT
 The MIT License (MIT)
 
-Copyright (c) 2016 Arvid Gerstmann.
-
+Copyright (c) 2014-2021 Arvid Gerstmann, the tldr-pages team and contributors.
```
Comment (zlatanvasovic) [inline man/tldr.1:47]:
```suggestion
Copyright (c) 2016 Arvid Gerstmann.
Copyright (c) 2016-2021 the tldr-pages team and contributors.
```

**Review by zlatanvasovic — CHANGES_REQUESTED** [review]

File: man/tldr.1 (hunk 2 above)
Comment (bl-ue) [reply man/tldr.1:47]: You think we should?  (cc @Leandros 😅)

Comment (sbrl) [reply man/tldr.1:47]: We really need to wait for them to reply. Perhaps someone could send an email to them?

Comment (SethFalco) [reply man/tldr.1:47]: @sbrl 

This was shared in Matrix lately, so just wanted to drop my thoughts here.

Reference: https://matrix.to/#/!zXiOpjSkFTvtMpsenJ:gitter.im/$WRhbUxjhliYZbRx4cml-JAuysKad2QyfJvZydecTtVQ?via=gitter.im&via=matrix.org&via=one.ems.host

To the best of my knowledge, though I am not a lawyer, there is no reason we would need their permission. That's the whole point of using an open-source license to redistribute work.

It's true that we must preserve the author's copyright notice as declared in the license.

Here is a source for the sake of it:

> You should not change or remove someone else’s copyright notice unless they have expressly (in writing) permitted you to do so. This includes third parties’ notices in pre-existing code.
> 
> — https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects

However, it's normal and expected to add new copyright notices when new contributors make changes. If anything, this unfortunately isn't done enough.

The original PR was modifying the existing notice, so I agree that was wrong. However, appending a new line that denotes the new copyright is fine. We must not **change** or **remove** the existing copyright notice. We can ofc, **append** another copyright notice after existing ones, if non-trivial (copyrightable) changes were made and adhere to the license.

Comment (sbrl) [reply man/tldr.1:47]: Okay, this makes sense @SethFalco. Will give my approval. Sorry this has taken so long!

**Comment by sbrl** [comment]
Ummm there doesn't seem to be an option for me to reopen this. Something seems to be wrong with @github-actions bot!

**Comment by sbrl** [comment]
Merged #109, which merges this PR.

**Pull request closed without merging**